webapp/.env.test

```diff
@@ -2,3 +2,5 @@
 KERNEL_CLASS='App\Kernel'
 APP_SECRET='$ecretf0rt3st'
 SYMFONY_DEPRECATIONS_HELPER=999999
+PANTHER_APP_ENV=panther
```

Comment:
out of curiosity: what is this?

Reply:
Panther is a web browser for testing. It came with one of the recipes but we can drop it

Reply:
I don't care, was mostly curious